src/main/java/org/sonarqube/gradle/AndroidUtils.java

```diff
@@ -101,6 +108,49 @@ private static void configureForAndroid(Project project, BaseVariant variant, Ma
     }
   }
 
+  private static void configureTestReports(Project project, BaseVariant variant, Map<String, Object> map) {
+    if (!(variant instanceof TestedVariant)) {
+      return;
+    }
+
+    if (GradleVersion.current().compareTo(GradleVersion.version("6.0")) < 0) {
+      // API to get task variant name is not available
+      return;
+    }
+
+    List<DirectoryProperty> directories = new LinkedList<>();
+
+    // junit tests
+    UnitTestVariant unitTestVariant = ((TestedVariant) variant).getUnitTestVariant();
+    if (unitTestVariant != null) {
+      directories.addAll(project.getTasks().withType(AndroidUnitTest.class).stream()
+        .filter(task -> unitTestVariant.getName().equals(task.getVariantName()))
+        .map(task -> task.getReports().getJunitXml().getOutputLocation())
+        .collect(Collectors.toList()));
+    }
+
+    // instrumentation tests
+    TestVariant testVariant = ((TestedVariant) variant).getTestVariant();
+    if (testVariant != null) {
+      directories.addAll(project.getTasks().withType(DeviceProviderInstrumentTestTask.class).stream()
+        .filter(t -> testVariant.getName().equals(t.getVariantName()) && t.getReportsDir().isPresent())
+        .map(DeviceProviderInstrumentTestTask::getReportsDir)
+        .collect(Collectors.toList()));
+    }
+
+    if (directories.isEmpty()) {
+      return;
+    }
+
+    String value = directories.stream()
+      .filter(Provider::isPresent)
+      .map(d -> d.get().getAsFile())
+      .filter(file -> file.isDirectory() && file.list().length > 0)
+      .map(File::getAbsolutePath)
+      .collect(Collectors.joining(","));
```

Comment:
We could put a list of files instead, it is resolve later by : org/sonarqube/gradle/SonarPropertyComputer.java:173

Reply:
ok, I'll fix

Reply:
actually since it's multiple files, I don't think that would work

Reply:
but I'll use `getEscapedFilePath`

Reply:
Multiple file is fine normally since org.sonarqube.gradle.SonarPropertyComputer#convertValue support List<File>

Reply:
ok, I'll try